client/src/components/NewAdmin.js

```diff
@@ -39,7 +39,7 @@ const NewAdmin = ({ setCreateMode }) => {
           alert("New account created successfully");
           setCreateMode(false);
 		} else if (res.status === 400) {
-			alert("an user with this email address already exists")
+			alert("an user with this username or email address already exists")
```

Comment:
Perhaps you wanna say 

'alert("a user with this username or email address already exists")'